{{cookiecutter.project_slug}}/android/fastlane/Fastfile

```diff
@@ -94,24 +99,21 @@ end
 def get_appetize_public_key(platform, base_name)
   return ENV['APPETIZE_PUBLIC_KEY'] unless ENV['APPETIZE_PUBLIC_KEY'].nil?
 
-  response = HTTParty.get("https://#{ENV['APPETIZE_API_TOKEN']}@api.appetize.io/v1/apps")
+  response =
+    HTTParty.get("https://#{ENV['APPETIZE_API_TOKEN']}@api.appetize.io/v1/apps")
 
-  if response.success?
-    json = JSON.parse(response.body)
+  return nil unless response.success?
 
-    # Find branch name in notes
-    json["data"].each do |value|
-      if value["note"] == base_name && value["platform"] == platform
-        return value["publicKey"]
-      end
-    end
+  item = JSON.parse(response.body)["data"].find do |app_app|
+    app_app['note'] == base_name && app_app['platform'] == platform
   end
 
-  nil
+  item['publicKey']
```

Comment:
If this doesn't find a match it will throw an exception. Previously it would return without an exception.

Reply:
I thought not finding it would be a pretty big deal, so left it out
will update

Reply:
It has been so long since I've written any Ruby I forgot unset hash values give you nil.

Reply:
I'm ashamed I was the one who hasn't remembered it initially and said I'd look into it 😊